packages/expo-brightness/ios/BrightnessEvents.swift

```diff
@@ -0,0 +1,3 @@
+enum BrightnessEvents {
+  static let didChange = "Expo.brightnessDidChange"
```

Comment:
Since this is just one value, I think you can just move it to `BrightnessModule.swift` as a top level variable 😉